README.md

```diff
@@ -12,7 +12,7 @@ If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to
 
 Instant messaging server matrix network.
 
-Yunohost chatroom with matrix : [https://riot.im/app/#/room/#yunohost:matrix.org](https://riot.im/app/#/room/#yunohost:matrix.org)
+YunoHost chatroom with Matrix : [https://matrix.to/#/#yunohost:libera.chat](https://matrix.to/#/#yunohost:libera.chat)
```

Comment:
```suggestion
YunoHost chatroom with Matrix : [https://matrix.to/#/#yunohost:matrix.org](https://matrix.to/#/#yunohost:matrix.org)
```

Well, the Matrix bridges got fixed and we Matrix users should now use the `#yunohost:matrix.org` room.